awx/main/models/inventory.py

```diff
@@ -927,6 +927,11 @@ class Meta:
         blank=True,
         default=1,
     )
+    limit = models.TextField(
+        blank=True,
+        default='',
+        help_text=_("Enter host, group or pattern match"),
```

Comment:
```
 further limit selected hosts to an additional pattern
 ```
 this is the help text from the ansible-playbook command

Reply:
You want me to mirror the help text from playbook?